js/src/apbct-public--5--external-forms.js

```diff
@@ -376,3 +378,102 @@ function sendAjaxCheckingFormData(form, prev, formOriginal) {
         }
     );
 }
+
+function catchDinamicRenderedForm() {
+    let forms = document.getElementsByTagName('form')
+
+    catchDinamicRenderedFormHandler(forms)
+
+    const frames = document.getElementsByTagName('iframe');
+    if ( frames.length > 0 ) {
+        for ( let j = 0; j < frames.length; j++ ) {
+            if ( frames[j].contentDocument == null ) {
+                continue;
+            }
+
+            const iframeForms = frames[j].contentDocument.forms;
+
+            if ( iframeForms.length === 0 ) {
+                return;
+            }
+
+            catchDinamicRenderedFormHandler(iframeForms, frames[j].contentDocument)
+        }
+    }
+}
+
+function catchDinamicRenderedFormHandler(forms, documentObject = document) {
+    let neededFormIds = [];
+    for (let form of forms) {
+        if (form.id.indexOf('hsForm') !== -1) {
+            neededFormIds.push(form.id)
+        }
+    }
+
+    for (let formId of neededFormIds) {
+        let form = documentObject.getElementById(formId);
+        form.onsubmit_prev = form.onsubmit;
```

Comment:
The object `onsubmit_prev` is already defined by visible fields logic. Please, rename this.

Reply:
https://github.com/CleanTalk/wordpress-antispam/pull/229/commits/3d1533010a866b4faa09dee70e0e0fd14384ceb6